### ISSUE TITLE
Is population data without replication suitable for rMATs?

### ISSUE THREAD

**Issue by qingzengLi** [Xinglab/rmats-turbo#356]
Thanks for providing such an excellent tool.I have a question for you
I have some RNA-seq data of different genotype materials. They are divided into treatment groups and control groups, but no replication. I want to know whether my data are suitable for rMATs.
Thanks a lot!

**Comment by EricKutschera** [comment]
Yes, that seems fine. It sounds like you have two sample groups, treatment and control, which is what rMATS expects. rMATS checks if there is a significant difference in isoform proportion between the two sample groups given as --b1 and --b2

**Comment by qingzengLi** [comment]
Wow,Thank you for your prompt reply,The materials in both groups are the same, should I use paired-stats mode?

**Comment by EricKutschera** [comment]
Here's the README section on the paired stats model: https://github.com/Xinglab/rmats-turbo/tree/v4.2.0#using-the-paired-stats-model

If each input file in the treatment group has a corresponding input file in the control group then you can use `--paired-stats`. The default statistical model can also be used and it should be much faster. You might try running the default model first to get initial results since --paired-stats could take a while

**Comment by qingzengLi** [comment]
OK,Thank you for you reply.
